packages/logging/src/servicies/logger/logger.service.ts

```diff
@@ -16,7 +16,7 @@ export class LoggerService implements Logger {
 
   private initPino(bindings?: Bindings): Logger {
     let options: LoggerOptions = {
-      level: config.LOG_LEVEL,
+      level: bindings?.logLevel || config.LOG_LEVEL,
```

Comment:
bindings is what goes to log object with every log, config is used to parametrize pino and logger. so the interface should be smth like

```ts
constructor(options?: LoggerOptions & { context: string })
```

this way existing client code wouldn't break. In the init though we'd need to put context into bindings